robotbase/src/main/java/frc/robot/subsystems/PhotonVisionCamera.java

```diff
@@ -35,6 +28,7 @@ public class PhotonVisionCamera extends SubsystemBase {
   protected PhotonPoseEstimator m_poseEstimator;
   protected final Transform3d ROBOT_TO_CAMERA_TRANSFORM; // if this changes, we have bigger issues.
   protected final double HEAD_ON_TOLERANCE;
+  protected static boolean s_connectionLost;
```

Comment:
This should be a member variable.

Reply:
its static so we dont print the warning message multiple times, one for each camera. Do you just want me to change the name of it to start with m_ instead of s_?